phac-nml/irida-next
# Update sidebar & breadcrumbs when editing group

## What does this PR do and why?
Updates the group name in the layout sidebar and breadcrumbs when a group name is successfully updated.
Pulled from [#192](https://github.com/phac-nml/irida-next/pull/192).

## Screenshots or screen recordings
![Screenshot from 2023-09-25 14-41-59](https://github.com/phac-nml/irida-next/assets/325703/bd18331f-b9e3-43fb-99b4-9ac8bfe700ce)

## How to set up and validate locally
1. Navigate to a Group
2. Click the `Settings` link in the sidebar
3. In the `Group name and description` form set `Group Name` to `a`
4. Click `Update group`
5. See that the form displays an error message and the sidebar and breadcrumbs are not updated.
6. Change the `Group Name` to `New group name`.
7. Click `Update group`
8. See a success message and that the sidebar an breadcrumbs have changed to the new name.

Note: Verify that change path refreshes the page to the new URL as expected.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
